amivapi/auth/sessions.py

```diff
@@ -38,16 +38,14 @@ def create_user_lookup_filter(self, user_id):
 
 sessiondomain = {
     'sessions': {
-        'description': {
-            'general': "A session is used to authenticate a user after he "
-            "provided login data. A POST to /session will return a token you "
-            "can use in an Authorization header: token <yourtoken>",
-            'methods': {
-                'POST': "Login and aquire a login token. Post the fields "
-                "'username' and 'password', the response will contain the "
-                "token. 'username' can be either nethz, mail, or user_id",
-                'GET': "Check token(s)."}
-        },
+        'description': "A session is used to authenticate a user after he "
+        "provided login data. A POST to /session will return a token you can "
+        "use in an Authorization header: token <yourtoken>",
+        # 'methods': {
+        #    'POST': "Login and aquire a login token. Post the fields "
+        #    "'username' and 'password', the response will contain the "
+        #    "token. 'username' can be either nethz, mail, or user_id",
+        #    'GET': "Check token(s)."}
```

Comment:
Is this documentation lost now? If yes, can we add it somewhere else?
If no, can we remove this comment?